### ISSUE TITLE
HD 143006 Imaging Tutorial Part 3

### ISSUE THREAD

**Issue by iancze** [MPoL-dev/MPoL#63]
An overview of the HD 143006 imaging series is provided in #25

Part III is meant to be the "production-ready" version of a MPoL imaging script. The idea is to use computational resources to do hyperparameter sweeps. One example is using [Ray Tune](https://docs.ray.io/en/master/tune/index.html), though we might also want to suggest alternatives.

* Explore "production-ready" scripting layouts (no Jupyter notebooks). Some useful code and directory structures [here](https://github.com/pytorch/examples/tree/master/fast_neural_style). 
* We have a starting point in the [examples/HD143006](https://github.com/MPoL-dev/MPoL/tree/main/examples/HD143006) subdirectory. These need to be improved to be more generalizable (no hardcoded paths) similar to the examples in the Pytorch directory. We also need to document how to visualize loss, etc. with tensorboard.
~~* Explore thorough hyperparameter testing with Ray Tune.~~
* Develop new priors if necessary to achieve a good image

This "tutorial" document might not actually be a *.py file -> Jupytext -> ipynb like the other tutorials. Rather, it might just be a regular *.rst file with text describing how one might go about running the each of the steps. If necessary, we can include a final notebook that uses the optimal parameters to visualize the best-fit image.

**Comment by jeffjennings** [comment]
Suggest adding a step demonstrating 'pre-training' to a dirty image (#206) to this tutorial.

**Comment by jeffjennings** [comment]
Suggest adding mention of use of a learning rate scheduler (#207) to this tutorial.

**Comment by jeffjennings** [comment]
Suggest adding the figure generated by #208 to this tutorial.
